script.js

```diff
@@ -0,0 +1,144 @@
+const todoForm = document.querySelector('.inputForm');
+const todoInput = todoForm.querySelector('input');
+const todoList = document.querySelector('.toDos');
+const doneList = document.querySelector('.dones');
+
+const saveList = 'savedList';
+let savedList = [];
+
+const saveFinishedList = 'savedFinList';
+let savedFinList = [];
+
+loadList();
+
+// To get the input from the form by adding an
+// event listener for submit button
+todoForm.addEventListener('submit', function (e) {
+  e.preventDefault();
+
+  const input = this.inputText;
+  const inputValue = input.value;
```

Comment:
event에 대한 함수니까 이런식으로 표현할수도 있을것 같아요!
```suggestion
  const input = this.inputText;
  const inputValue = e.target.value;
```